content/en/docs/about/use-cases.md

```diff
@@ -5,11 +5,6 @@ weight = 1
 aliases = ["/docs/", "/docs//", "/docs/usecases/"]
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
-
 
 The end goal of every organization is to have their machine learning (ML) model successfully running in production  and generating value to the business. But what does it take to reach that point? Before a model ends up in production, there are potentially many steps required to build and deploy an ML model: data loading, verification, splitting, processing, feature engineering, model training and verification, hyperparameter tuning, and model serving. In addition, ML models can require more observation than traditional applications, because your data inputs can drift over time. Manually rebuilding models and data sets is time consuming and error prone. To simplify these requirements and challenges, we introduce Kubeflow.
 
```

Comment:
A few suggestions:

- Three paragraphs for readability/scannability:
  1) KF
  2) KF and k8s
  3) CUJs
- First complex sentence -> two shorter ones
- "top researchers" -> "researchers" (perhaps "top" is subjective)
- "~~This means~ your ... researchers have more time" -> "Your researchers can have more time..." (hopefully this frees up their time :))
- "i.e." - ":"

```
Kubeflow is a scalable, portable, distributed ML platform that runs on 
Kubernetes. This means that all capabilities of Kubernetes are available to a 
Kubeflow user. With Kubeflow, you can manage the entire AI organization at scale 
and still be able to maintain the same quality of control. 

Kubeflow and Kubernetes provide consistent and efficient operations and 
optimized infrastructure. Your researchers can have more time to focus on the 
valuable tasks of developing domain specific intellectual property rather than 
debugging DevOps configuration issues. 

Kubeflow’s core and ecosystem critical user journeys (CUJs) provide software 
solutions for end-to-end workflows: build, train and deploy and/or develop a 
model and create, run and explore a pipeline.
```